tasks/fix-cat2.yml

```diff
@@ -1380,24 +1380,29 @@
   tags:
       - RHEL-07-030350
 
-- name: "MEDIUM | RHEL-07-030360 | All privileged function executions must be audited."
+- name: "MEDIUM | RHEL-07-030360 | PATCH | The Red Hat Enterprise Linux operating system must audit all executions of privileged functions."
   block:
-      - name: "MEDIUM | RHEL-07-030360 | AUDIT | All privileged function executions must be audited. (find suid/sgid programs)"
-        command: find {{ rhel7stig_local_mounts | join(' ') }} -xdev -type f ( -perm -4000 -o -perm -2000 )
-        check_mode: no
-        changed_when: no
-        register: rhel_07_030360_audit
+      - name: "Delete obsolete suid/sgid rules"
+        file:
+            path: "/etc/audit/rules.d/rhel7stig_suid_sgid_commands.rules"
+            state: absent
 
-      - name: "MEDIUM | RHEL-07-030360 | All privileged function executions must be audited."
+      - name: "Audit rules for execve calls"
         lineinfile:
-            path: "/etc/audit/rules.d/rhel7stig_suid_sgid_commands.rules"
+            path: "/etc/audit/rules.d/rhel7stig_execve_calls.rules"
             create: yes
             owner: root
             group: root
             mode: 0600
-            line: "-a always,exit -F path={{ item }} -F perm=x -F auid>=1000 -F auid!=4294967295 -k setuid/setgid"
+            line: "{{ item }}"
             state: present
-        with_items: "{{ rhel_07_030360_audit.stdout_lines }}"
+        with_items:
```

Comment:
I wonder if maybe we should start introducing the usage of `loop` instead of `with_items`. It's not terribly important at the moment, maybe it can just be part of a larger modernization refactor to keep in line with changes in Ansible upstream

Reply:
Would break ansible 2.4. with_<lookup> is not deprecated yet.

Reply:
Ansible 2.4 is EOL https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#release-status